tgui/packages/tgui/interfaces/AlertComputer.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button, Flex, Icon, Table } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface AlertComputerData {
+  alerts: AlertData[];
+}
+
+interface AlertData {
+  zone: string;
+  area_ckey: string;
+  atmos: number;
+  fire: number;
+  power: number;
+  motion: number;
+}
+
+export const AlertComputer = () => {
+  const { data } = useBackend<AlertComputerData>();
+  const { alerts } = data;
+  return (
+    <Window title="Current Alerts" width={600} height={420}>
+      <Window.Content scrollable>
+        <Table>
+          <Table.Row header fontSize="1.1em" className="candystripe">
+            <Table.Cell header width="200px">
+              Area
+            </Table.Cell>
+            <Table.Cell header width="100px">
+              <Icon name="wind" mr="5px" />
+              Air
+            </Table.Cell>
+            <Table.Cell header collapsing width="100px">
+              <Icon name="fire" mr="5px" />
+              Fire
+            </Table.Cell>
+            <Table.Cell header collapsing width="100px">
+              <Icon name="bolt" mr="5px" />
+              Power
+            </Table.Cell>
+          </Table.Row>
+          {Object.values(alerts).map((value, index) => {
```

Comment:
Per typing of AlertComputerData, `alerts` is an array so doing `Object.values` is not needed and you can just do `alerts.map(...)`. Is the typing wrong, or is this part wrong?